cdk-infra/lib/constructs/worker/worker-construct.ts

```diff
@@ -103,6 +116,7 @@ export class WorkerConstruct extends Construct {
       taskDefinition,
       desiredCount: 5,
       minHealthyPercent: 100,
+      maxHealthyPercent: 200,
```

Comment:
Allows up to 10 tasks per service. This allows for existing jobs to be processed in the event an update occurs.